pipelines/batch/setup_eval_job.py

```diff
@@ -97,41 +97,46 @@ def main(
         mount_pairs=[
             {
                 "source": "nssp-etl",
-                "target": "/pyrenew-hew/nssp_demo/nssp-etl",
+                "target": "/pyrenew-hew/nssp-etl",
             },
             {
                 "source": "nssp-archival-vintages",
-                "target": "/pyrenew-hew/nssp_demo/nssp-archival-vintages",
+                "target": "/pyrenew-hew/nssp-archival-vintages",
             },
             {
                 "source": "prod-param-estimates",
-                "target": "/pyrenew-hew/nssp_demo/params",
+                "target": "/pyrenew-hew/params",
             },
             {
-                "source": "pyrenew-test-output",
-                "target": "/pyrenew-hew/nssp_demo/private_data",
+                "source": "pyrenew-hew-prod-output",
+                "target": "/pyrenew-hew/output",
+            },
+            {
+                "source": "pyrenew-hew-config",
+                "target": "/pyrenew-hew/config",
             },
         ],
     )
 
     base_call = (
         "/bin/bash -c '"
-        "python nssp_demo/forecast_state.py "
+        "python pipelines/forecast_state.py "
         "--disease {disease} "
         "--state {state} "
         "--n-training-days 365 "
         "--n-warmup 1000 "
         "--n-samples 500 "
-        "--facility-level-nssp-data-dir nssp_demo/nssp-etl/gold "
+        "--facility-level-nssp-data-dir nssp-etl/gold "
         "--state-level-nssp-data-dir "
-        "nssp_demo/nssp-archival-vintages/gold "
-        "--param-data-dir nssp_demo/params "
-        "--output-data-dir nssp_demo/private_data "
+        "nssp-archival-vintages/gold "
+        "--param-data-dir params "
+        "--output-data-dir output "
+        "--priors-path config/eval_priors.py "
```

Comment:
^ These live in blob storage, within the storage bucket ("container") that will be mounted within the pyrenew (dockerlike) container under the name the name `config`

Reply:
The `default_priors.py` file is to make local runs / testing easier. You'd just point `forecast_state.py` at it instead of at, say, `eval_priors.py` in blob storage.

Reply:
Runs themselves save their `priors.py` file as part of the model run directory, which I think is the most important reproducibility guarantee

https://github.com/CDCgov/pyrenew-hew/blob/ae9b8676f6fd2bcec35851c9d1a9559ae5e39d04/pipelines/forecast_state.py#L177-L178

We should also have model run dirs save the commit hash, container version, etc metadata, but that's out of scope here imo

Reply:
Ah, right. Thanks for clarifying.